docs/pull_request_guidelines.md

```diff
@@ -0,0 +1,35 @@
+## Creating PR
+1. Name [username]/branchname 
+    1. Branch name should be all lowercase
+    2. Words separated by “-”
+2. Code should address only one issue ideally, make a separate PR for each task
+3. Description 
+    1. Clear explanation of issues solvedDescribe why and how when appropriate
```

Comment:
typo?

Reply:
yep, google doc -> markdown is not an automated process... Will fix